### ISSUE TITLE
Logo in README can't be fully seen

### ISSUE THREAD

**Issue by KalobTaulien** [wagtail/wagtail#8160]
![image](https://user-images.githubusercontent.com/4743971/158834123-154586c2-5171-485a-bcc1-b195f5bd9f7e.png)

![image](https://user-images.githubusercontent.com/4743971/158834256-408ed52d-0d6e-4d1d-9387-23f24dfc7456.png)

The new logo works for GitHub's dark mode, but not with GitHub's light mode. 

We may want to create a new logo with our own background color rather than using a transparent background.

**Comment by PaarthAgarwal** [comment]
It is working fine for me @KalobTaulien 
Tested on: edge browser (windows and linux)
Demo:
https://user-images.githubusercontent.com/86092410/158835863-57bebe7f-fcdb-4775-bfa0-d533fb1e512d.mp4

**Comment by lb-** [comment]
@KalobTaulien what browser are you using?

I also tested this when merging and have tested again now and it worked for my on my phone and desktop browsers. 

![Screenshot_20220318-050954](https://user-images.githubusercontent.com/1396140/158878129-77fe405b-14fc-40e4-8166-49e68a4113db.png)

**Comment by amrht** [comment]
Logo is not getting changed in dark mode, I  am proposing that we should use logo with our own background.
![Screenshot (60)](https://user-images.githubusercontent.com/73494006/158878293-51d461d5-d7d0-479a-ad00-d775e21e4df6.png)

**Comment by lb-** [comment]
Odd. That screenshot is showing the opposite issue. 

Here is the PR for reference https://github.com/wagtail/wagtail/pull/8151
And the original issue https://github.com/wagtail/wagtail/issues/8135

**Comment by PaarthAgarwal** [comment]
> Odd. That screenshot is showing the opposite issue.
> 
> Here is the PR for reference #8151 And the original issue #8135
> 
> For context - the solution implemented is based on GitHub's docs
> 
> https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to
> 
> Current theory - maybe we need to use markdown syntax.
> 
> Also. Just realised that this may not show great in PyPi

This issue can also be caused if one chooses dark variants in the Dag theme(recognized as light) or light variants in Night theme(recognized as dark) in the GitHub appearances tab.
Like these:
![image](https://user-images.githubusercontent.com/86092410/158882814-575d6d7a-f72d-4075-b499-c8592bb4ee09.png)
Though its an odd usage case still you can replicate the buggy behavior by trying those variants.

**Comment by amrht** [comment]
In my browser, Logo is not changing in any variant. Also, tried extensions and github settings both, but it is same.

**Comment by PaarthAgarwal** [comment]
@amrht which browser are you using?

**Comment by lb-** [comment]
I think we may need to revert the change. Or explore an image that works on both. 

Maybe not a full background but more like a white glow that sits behind the original logo. So in dark mode it is readable. 

Also - the logo structure as is will cause issues in PyPi

**Comment by amrht** [comment]
I think We should use a logo with our own background as mentioned earlier by someone, it will be better for now.

**Comment by amrht** [comment]
@PaarthAgarwal Chrome Version 99.0.4844.74 (Official Build) (64-bit)

**Comment by KalobTaulien** [comment]
I am using Chrome as well. 

There are two scenarios: 
1. When my OS is set to "light mode", it shows the black logo and works as expected. 
2. When my OS is set to "dark mode", it shows the white logo as expected. But when I'm signed into GitHub using light mode, it shows up with the white logo on a white background.

**Comment by lb-** [comment]
Update. Pypi does not use the readme.md file so the concern I had above is not valid. 

The issue for GitHub loading still exists though.

**Comment by Scotchester** [comment]
:eyes: https://github.blog/changelog/2022-05-19-specify-theme-context-for-images-in-markdown-beta/

**Comment by thibaudcolas** [comment]
This seems to be working for me with both OS-level dark and light mode, as well as all of GitHub’s themes.

Unfortunately it fails in Windows High Contrast mode (dark theme):

<img width="520" alt="whcm-logo" src="https://user-images.githubusercontent.com/877585/185264231-4716c8da-0069-45e4-ba29-f73cc434014d.png">

So – we can follow GitHub’s recommended approach for this, and we also need our images to have a plain background so it works for WHCM users.

**Comment by thibaudcolas** [comment]
Tagging this as a _good first issue_. I believe the only work needed is for someone to edit our two logos to add a background.

- https://github.com/wagtail/wagtail/blob/main/.github/wagtail.svg should have a plain white background rather than transparent
- https://github.com/wagtail/wagtail/blob/main/.github/wagtail-inverse.svg should have a plain black background rather than transparent

**Comment by benitaanawonah** [comment]
Hi @thibaudcolas, I'm interested in working on this, can you assign it to me?

**Comment by lb-** [comment]
@benitaanawonah  - we do not assign issues to people (outside of core team) - just mention you are starting to work on this and that's all that is needed.

Please read the comments above - this works in most cases but not in Windows High Contrast mode.

Have fun - thank you.

**Comment by xomcodes** [comment]
Hello @lb- @thibaudcolas I would love to work on this if @benitaanawonah  is not working on it

**Comment by Scotchester** [comment]
@xomcodes It's only been 14 hours since they expressed their interest. Let's give them some time to work 🙂

**Comment by xomcodes** [comment]
Okay. That's true

**Comment by benitaanawonah** [comment]
@lb- Okay, Got it. Working on it.

**Comment by thibaudcolas** [comment]
Great! If you want to try to reproduce the issue, here is how to simulate high-contrast mode in Chrome: https://developer.chrome.com/blog/new-in-devtools-98/#forced-colors

Though again really all that’s needed is to add the background colors.

**Comment by lb-** [comment]
If you or anyone needs to edit svgs. Inkscape is good for this. 
https://inkscape.org/

**Comment by benitaanawonah** [comment]
@lb- I've made the changes and I’m currently testing, so I'll be sending my pull request soon
